src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -33,6 +33,16 @@ public class HttpWorkerOptions : IOptionsFormatter
 
         public TimeSpan InitializationTimeout { get; set; } = TimeSpan.FromSeconds(30);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether custom routes are enabled.
+        /// </summary>
+        public bool CustomRoutesEnabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets route mapping for a HTTP worker.
```

Comment:
nit: This description reflects the state today, but the `CustomHandlerHttpOptions` wouldn't be limited to route mapping (today, it already handles authorization level configuration).